extensions/blocks/calendly/calendly.php

```diff
@@ -88,41 +89,54 @@ function load_assets( $attr, $content ) {
 
 		// Render deprecated version of Calendly block if needed. New markup block button class before rendering here.
 		if ( false === strpos( $content, 'wp-block-jetpack-button' ) ) {
-			$content = deprecated_render_button_v1( $attr, $block_id, $classes, $url );
+			$content = deprecated_render_button_v1( $attr, $html_id, $classes, $url );
 		} else {
-			$content = str_replace( 'calendly-widget-id', esc_attr( $block_id ), $content );
+			$content = str_replace( 'calendly-widget-id', esc_attr( $html_id ), $content );
 			$content = str_replace( $base_url, $url, $content );
 		}
 
 		if ( ! $is_amp_request ) {
 			wp_add_inline_script(
 				'jetpack-calendly-external-js',
-				sprintf( "calendly_attach_link_events( '%s' )", esc_js( $block_id ) )
+				sprintf( "calendly_attach_link_events( '%s' )", esc_js( $html_id ) )
 			);
 		}
 	} else { // Inline style.
 		if ( $is_amp_request ) {
 			$content = sprintf(
 				'<div class="%1$s" id="%2$s"><a href="%3$s" role="button" target="_blank">%4$s</a></div>',
 				esc_attr( Blocks::classes( FEATURE_NAME, $attr ) ),
-				esc_attr( $block_id ),
+				esc_attr( $html_id ),
 				esc_js( $url ),
 				wp_kses_post( get_attribute( $attr, 'submitButtonText' ) )
 			);
 		} else {
-			$content = sprintf(
+			$content          = sprintf(
 				'<div class="%1$s" id="%2$s"></div>',
 				esc_attr( $classes ),
-				esc_attr( $block_id )
+				esc_attr( $html_id )
 			);
-			$script  = <<<JS_END
-Calendly.initInlineWidget({
-	url: '%s',
-	parentElement: document.getElementById('%s'),
-	inlineStyles: false,
-});
+			$escaped_block_id = esc_js( $block_id );
+			$script           = <<<JS_END
+// Call function with block id in name
+// to allow for multiple widgets.
+function jetpackInitCalendly{$escaped_block_id}() {
+	Calendly.initInlineWidget({
+		url: '%s',
+		parentElement: document.getElementById('%s'),
+		inlineStyles: false,
+	});
+};
+// For P2s only: wait until after o2 has
+// replaced main#content to initialize widget.
```

Comment:
Minor suggestion. We should use the [multiline format](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments) defined by WordPress core - PHP documentation standards.

```suggestion
/*
 * For P2s only: wait until after o2 has
 * replaced `main#content` to initialize widget.
 */
```

Reply:
Just realized that it's the javascript code, generated in the dynamically in the backend. Feel free to ignore these comments.